egs/chime6/s5_track2/local/decode.sh

```diff
@@ -8,11 +8,11 @@
 # Apache 2.0
 
 # Begin configuration section.
-nj=8
+nj=10
 decode_nj=10
 stage=0
 sad_stage=0
-diarizer_stage=0
+diarizer_stage=1
```

Comment:
this is correct?

Reply:
Stage 0 basically keeps Array U06 and discards all others, so it's a matter of choice. But yeah, for the baseline, we should start at stage 0. I'll fix it with a few other things in my next commit. This PR is still WIP.